src/Bundler/Runner/Exception/SocketException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * @copyright 2017 Hostnet B.V.
+ */
+declare(strict_types=1);
+
+namespace Hostnet\Component\Resolver\Bundler\Runner\Exception;
+
+class SocketException extends \Exception
```

Comment:
Can you add some doc what this exception is used for.

extend `\RuntimeException`?